rosbridge_server/launch/rosbridge_websocket_client.launch

```diff
@@ -10,6 +11,7 @@
       <param name="certfile" value="$(arg certfile)" />
       <param name="keyfile" value="$(arg keyfile)" />
       <param name="authenticate" value="$(arg authenticate)" />
+      <param name="user_auth" value="$(arg enable_authentication)" />
```

Comment:
@Shokman this is not compatible with the change in rosbrdige_websocket.launch, the parameter names don't match